### ISSUE TITLE
instagram post text based on content used

### ISSUE THREAD

**Issue by phivk** [unfoldcollective/Apprentice-of-Okolicne#92]
- [x] figure out generative 'recipe'
- [x] add emojis and hashtags to parts.json (@phillchill)
- [ ] implement in code

**Comment by phivk** [comment]
working on template texts: https://docs.google.com/document/d/1Lo80LszvKFxYqiyvit9_ZW6EyiFeoUYcjgJdeVyIv9A/edit?usp=sharing

**Comment by phivk** [comment]
@bertez some elements contain the same hashtags and emojis, it would be good to insert only a unique set without duplicates into the message text

**Comment by bertez** [comment]
Yes, I already do this for the repeated figures on the artwork list.

**Comment by phivk** [comment]
thinking out of the 🗃 ... every time a composition is added, we could update the instagram profile picture:

initial B
![icon-b](https://user-images.githubusercontent.com/902958/30152455-737af2e4-93b3-11e7-9879-6769dc252d4a.png)

initial P
![icon-p](https://user-images.githubusercontent.com/902958/30152456-737fabf4-93b3-11e7-98ed-c5df4e3a154f.png)

**Comment by phivk** [comment]
all emojis and hashtags are added on `feature/add-real-content`

**Comment by bertez** [comment]
@phillchill the idea of changing the profile pic on instagram is really good, but I think it will increase the complexity of the post-save process and more important, I cannot find information about the limits of the account endpoint so it could be rate limited.

**Comment by phivk** [comment]
OK, let's not do the profile pic...
